src/integTest/groovy/com/readytalk/LifecycleIntegTest.groovy

```diff
@@ -10,7 +10,7 @@ class LifecycleIntegTest extends IntegrationSpec {
       apply plugin: 'com.readytalk.ci'
       apply plugin: 'java'
 
-      buildEnv.isCI = false
+      buildEnv.ci = false
```

Comment:
Yeah, I agree with that. I don't think we should try to contort ourselves that much to support Java 6 projects. There are so many reasons to get off of Java 6 - lets do what we can to apply even more pressure there.